client/state/selectors/is-jetpack-cloud-eligible.js

```diff
@@ -1,7 +1,7 @@
 /**
  * Internal dependencies
  */
-import getSiteOptions from 'calypso/state/selectors/get-site-options';
+import getSiteOptions from 'calypso/state/sites/selectors/get-site-options';
```

Comment:
Should we be consistent and import all sites selectors as modules from `calypso/state/sites/selectors` instead of specifically passing the path to the selector file? The former would also be shorter FWIW.